src/routes/rules/components/RulePreviewDialog.vue

```diff
@@ -73,3 +73,10 @@ const preloadImage = () => {
   img.src = props.imageUrl;
 };
 </script>
+
+<style>
+.game-overlay {
+  width: 650px;
+  max-width: 90vw;
+}
```

Comment:
Can you send a couple screenshots how this looks on a device that's say 600px vs 1920px wide?

Reply:
600px wide:
![Captura de pantalla 2024-06-07 193449](https://github.com/cuttle-cards/cuttle/assets/87356920/e6304ef9-c8f9-44b5-bea8-5c2806e60c52)
1920px wide:
![Captura de pantalla 2024-06-07 193533](https://github.com/cuttle-cards/cuttle/assets/87356920/0f054938-cf54-40ca-9907-43ab1d3bce22)